src/java.base/share/classes/jdk/internal/loader/ClassLoaders.java

```diff
@@ -55,15 +55,20 @@ private ClassLoaders() { }
     private static final PlatformClassLoader PLATFORM_LOADER;
     private static final AppClassLoader APP_LOADER;
 
+    private static void setArchivedServicesCatalog(ArchivedClassLoaders archivedClassLoaders, ClassLoader loader) {
```

Comment:
I'd prefer if we could keep the code consistent if possible. In this case, the method declaration can be split over two lines to avoid one really one line in the file. Also can you add a one line // comment to be consistent with the other private methods.